components/Common/Sidebar/SidebarGroup/index.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import type { FC } from 'react';
+
+import SidebarItem from '@/components/Common/Sidebar/SidebarItem';
+
+import styles from './index.module.css';
+import type { ActiveItem, SidebarGroupType } from '..';
+
+type SideBarGroupProps = SidebarGroupType & {
+  activeItem?: ActiveItem;
+  setActiveItem: (item: ActiveItem) => void;
+};
```

Comment:
Component prop types can be simplified as follows:

```suggestion

type SidebarGroupProps = {
  groupName: string;
  items: ComponentProps<typeof SidebarItem>[];
  activeItem?: ComponentProps<typeof SidebarItem>;
};
```